10-inference.Rmd

```diff
@@ -210,41 +210,29 @@ the mean of the sample proportions.
 sample_estimates %>%
   summarise(mean = mean(sample_proportion))
 ```
-We notice that the sample proportions are centred around the population proportion value, `r round(mean(sample_estimates$sample_proportion), 2)`! In general, the mean of the distribution of $\hat{p}$ should be equal to $p$, which is good because that means the sample proportion is neither an overestimate nor an underestimate of the population proportion.
+We notice that the sample proportions are centred around the population proportion value, `r round(mean(sample_estimates$sample_proportion), 3)`! In general, the mean of the distribution of $\hat{p}$ should be equal to $p$, which is good because that means the sample proportion is neither an overestimate nor an underestimate of the population proportion.
 
-So what can we learn from this sampling distribution? This distribution tells us what we might expect from proportions from samples of size $40$ when our population proportion is 0.63. In practice, we usually don't know the proportion of our population, but if we can use what we know about the sampling distribution, we can use it to make inferences about our population when we only have a single sample. 
+So what can we learn from this sampling distribution? This distribution tells us what we might expect from proportions from samples of size $40$ when our population proportion is `r round(population_proportion,3)`. In practice, we usually don't know the proportion of our population, but if we can use what we know about the sampling distribution, we can use it to make inferences about our population when we only have a single sample. 
```

Comment:
"So what can we learn from this sampling distribution?"

isn't the text right before this telling us what we can learn from this sampling distribution?

"this distribution tells us what we might expect from proportions from samples of size 40 when ...."

that sentence doesn't really convey anything about what we might learn from the sampling distribution.

Honestly I might just suggest removing this paragraph, it doesn't do much